src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -15411,10 +15412,11 @@ namespace OutputReportTabular {
             Lights(iLight).SumTimeNotZeroCons = 0.;
             Lights(iLight).SumConsumption = 0.;
         }
-        for (iLight = 1; iLight <= NumExteriorLights; ++iLight) {
```

Comment:
this was moved to the clear_state() in EnergyPlusData.hh and then removed since it followed an ExteriorLights.deallocate()